README.md

```diff
@@ -66,22 +66,22 @@ Here are the main points we focus on:
 
 * Stability
 * Ease of use / Practicability
-* Playability (ingame content)
+* Playability (in-game content)
 * Customization
-* Community driven software (check our discord)
+* Community-driven software (check our discord)
 
-We also welcome new users (even non-english speaking users!) and help them learn / improve their skills (C++, SQL, Git, software collaboration, tutoring / wiki etc...).
+We also welcome new users (even non-English speaking users!) and help them learn/improve their skills (C++, SQL, Git, software collaboration, tutoring/wiki, etc...).
 
-Unlike other projects which focus more on the developer's side of things, we want users to be able to run their own server with as few troubles as possible. All of our contributors run their own private server (in local or not).
+Unlike other projects which focus more on the developer's side of things, we want users to be able to run their server with as few troubles as possible. All of our contributors run their private servers (local or public).
```

Comment:
same comment for "own"

i emphasize this because that's why people use azerothcore, to make their OWN thing

people are selfish :)

Reply:
Their own is basically just repeating the same word two times since it means the same in this context